plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeOutputTableHandle.java

```diff
@@ -45,7 +46,8 @@ public DeltaLakeOutputTableHandle(
             @JsonProperty("location") String location,
             @JsonProperty("checkpointInterval") Optional<Long> checkpointInterval,
             @JsonProperty("external") boolean external,
-            @JsonProperty("comment") Optional<String> comment)
+            @JsonProperty("comment") Optional<String> comment,
+            @JsonProperty("changeDataFeedEnabled") Optional<Boolean> changeDataFeedEnabled)
```

Comment:
I'd be in favour of using `boolean` instead of `Optional<Boolean>` here.

Reply:
Same thing for `io.trino.plugin.deltalake.DeltaLakeTableProperties#isChangeDataFeedEnabled`

Reply:
Initially i had boolean here, but that was problematic because the it added the property set to false even when user didn't set it, having possibility for null/lack of value is important in this case